.gitignore

```diff
@@ -24,4 +24,4 @@ test/screenshots
 test/videos
 test/dev.html
 yarn-debug.log
-yarn-error.log
+yarn-error.log
```

Comment:
looks like we are missing a newline at the end of this file, .npmrc, and .yarnrc so that these changes could be removed from this PR